src/store/configureStore.prod.js

```diff
@@ -15,7 +15,7 @@ import serviceReducer from 'reducers';
 import initialState from './initial_state';
 import {offlineConfig} from './helpers';
 
-export default function configureOfflineServiceStore(preloadedState, appReducer, userOfflineConfig, enableBuffer = true) {
+export default function configureOfflineServiceStore(preloadedState, appReducer, userOfflineConfig, getAppReducer, enableBuffer = true) {
```

Comment:
The prod and dev signatures for this function need to match, doesn't matter if that's not used here